ghdocs/Theming.md

```diff
@@ -0,0 +1,85 @@
+# Theming
+
+This document describes theming at a high level. For low level technical details, follow the links.
+
+
+## What is theming?
+
+Theming is a mechanism by which a consistent look and feel can be applied to all the components on a page. For now, this means sharing a color scheme across the entire page.
+o match this color scheme.
+
+
+## What's in a theme?
+A theme is defined by a simple collection of variables (which we call slots) with string values.
+For example, `themePrimary` by default is `"#0078d7"`, but it could easily be `"rgba(0, 120, 215)"`.
+
+
+## What are theme slots?
+
+There are two kinds of theme slots: Fabric palette slots and semantic slots.
+
+Fabric palette slots are documented in [IPalette.ts](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/styling/src/interfaces/IPalette.ts).
+These are descriptive slots, it gives you an idea of what the color is, but you decide how to use it.
+Most slots with color names (besides `white` and `black`), such as `yellow` and `yellowLight`, will remain a shade of yellow in all themes, useful in cases where color has meaning (such as for errors and warnings).
+Customizing Fabric palette slots allows broad-stroke changes to the overall color scheme.
+
+Semantic slots are documented in [ISemanticColors.ts](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/styling/src/interfaces/ISemanticColors.ts).
+These, on the other hand, are prescriptive slots, each slot having an intended use.
+This allows more targeted customizations.
+For example, you could change the color of light gray disabled controls without affecting the light gray used by list item hover/selection state.
```

Comment:
This feels confusing. I would relate the color to a specific disabled control, rather than keeping it so generic. So it would be more like "you can change the grey color of disabled buttons without impacting the grey used in a list item.